pkgs/tools/X11/xsecurelock/default.nix

```diff
@@ -19,6 +38,13 @@ stdenv.mkDerivation rec {
     apacheHttpd imagemagick pamtester
   ];
 
+  # Allow adding extra modules to the helper directory
+  postInstall =
+    let
+      copyHelper = path: "cp -v ${path} $out/libexec/xsecurelock/";
```

Comment:
The problem with doing this is we have to rebuild with every new module. Instead, I would recommend patching xsecurelock so that HELPER_PATH is ```/run/curren-system/sw``` that way modules can just go into systemPackages.

Reply:
Does that break when you don't have the package installed system-wide, though? Since it's a compile-time option, I think that means that if you're installing this using `nix-env -i`, `xsecurelock` won't be able to find any helpers since they'll be in `/nix/var/nix/profiles/`?